test-network/scripts/utils.sh

```diff
@@ -146,9 +198,26 @@ function printHelp() {
     println "   network.sh createChannel -c channelName"
     println "   network.sh deployCC -ccn basic -ccp ../asset-transfer-basic/chaincode-javascript/ -ccl javascript"
     println "   network.sh deployCC -ccn mychaincode -ccp ./user/mychaincode -ccv 1 -ccl javascript"
+    println
+    println " NOTE: Default settings can be changed in network.config"
   fi
 }
 
+function installPrereqs() {
+
+  infoln "installing prereqs"
+
+  FILE=../install-fabric.sh     
+  if [ ! -f $FILE ]; then
+    curl -sSLO https://raw.githubusercontent.com/hyperledger/fabric/main/scripts/install-fabric.sh && chmod +x install-fabric.sh
+    cp install-fabric.sh
```

Comment:
I think you intended to copy it to the parent directory fabric-samples?
```suggestion
    cp install-fabric.sh ..
```